code/LINDA/LINDA_fire.dm

```diff
@@ -26,13 +26,23 @@
 	if((exposed_temperature > PLASMA_MINIMUM_BURN_TEMPERATURE) && air_contents.toxins > 0.5)
 		igniting = 1
 
+	var/obj/effect/decal/cleanable/liquid_fuel/liquid = locate() in src
```

Comment:
I can't approve of any rampant modifications to LINDA code for the sake of a minor feature.

While not perfectly ideal, due to plasma overlays, the correct way to start fires is with the `atmos_spawn_air` proc.
It accepts a flag and amount, the way to make a self-sustaining plasma fire would be `src.atmos_spawn_air(SPAWN_HEAT|SPAWN_TOXINS, arbitrary amount of plasma over the 0.5 threshold)`